mythril/laser/ethereum/instructions.py

```diff
@@ -230,18 +230,18 @@ def mulmod_(self, global_state):
     @instruction
     def exp_(self, global_state):
         state = global_state.mstate
-        # we only implement 2 ** x
-        base, exponent = util.pop_bitvec(state), util.pop_bitvec(state)
 
+        base, exponent = util.pop_bitvec(state), util.pop_bitvec(state)
         if (type(base) != BitVecNumRef) or (type(exponent) != BitVecNumRef):
-            state.stack.append(BitVec("(" + str(simplify(base)) + ")^(" + str(simplify(exponent)) + ")", 256))
+            state.stack.append(BitVec("(" + str(simplify(base)) + ")**(" + str(simplify(exponent)) + ")", 256))
         elif base.as_long() == 2:
```

Comment:
Is this still necessary?

Reply:
I thought it would be faster for them to be separate but now I realized that this is all in python and that wouldn't be the case :stuck_out_tongue: